public/js/app/views/BookmarkletView.js

```diff
@@ -6,6 +6,18 @@ define(["app/app",
     templateName: 'bookmarklet',
     template: Ember.Handlebars.compile(tpl),
 
+    didInsertElement: function() {
+      var field = this.$("#bookmarklet-recipients")
+      field.select2({ width: '100%' })
+
+      // When the select2 became multiline it change bookmarklet height, but we can't
+      // handle this via CSS rules, so use JS to increase iframe size accordingly
+      // Only way to interact with the scripts outside the iframe is postMessage
+      field.on('change', function() {
+        window.parent.postMessage(window.document.documentElement.offsetHeight, '*')
```

Comment:
Since we're dealing with 3rd-party websites, is there a possibility we'll have conflicts with just sending a number? Like, if website does the same (sending and receiving a number), we'll have erroneous triggering with popup height going wild. Maybe it's worth to pass some sort of structure instead (like, `{type: 'freefeed-bookmarklet-height', value: 123}`) and check for it in the receiver?